src/app/public/modules/inline-delete/inline-delete.component.html

```diff
@@ -0,0 +1,41 @@
+<div
+  aria-describedby="sky-inline-delete-assistive-text"
+  class="sky-inline-delete sky-inline-delete-{{type}}"
+  role="alertdialog"
+  [attr.aria-label]="'skyux_inline_delete_confirm_deletion' | skyLibResources"
+>
+  <span
+   class="sky-inline-delete-assistive-text"
+   id="sky-inline-delete-assistive-text"
+  >
+    {{ 'skyux_inline_delete_assistive_text' | skyLibResources }}
+  </span>
+  <div *ngIf="pending"
+    class="sky-inline-delete-wait"
+  >
+    <sky-wait
+      [isWaiting]="true"
+    ></sky-wait>
+  </div>
+  <div
+    class="sky-inline-delete-content"
+    [@skyAnimationEmerge]="animationState"
```

Comment:
In the visual demo, I'm not seeing any animations... The inline-delete component just shows up immediately. What should I be seeing here?

Reply:
That is due to the `NoopAnimationsModule` being in `app-extras` to ensure that visual tests aren't taken mid-animation. We do that in a lot of our libraries. If you remove it locally then the animations will work.